### DISCUSSION TITLE
Advise for tags

### DISCUSSION THREAD

**Discussion by nmcrae85** [ESPresense/ESPresense#1087] [Q&A]
### Describe the bug

More advise. I’ve got some smart tags from Samsung which I know aren’t supported. However, I want to use something tag based for some car keys. What are other using that are known to work? Also cards (one that fit in a wallet). I’m based in the UK so things avalible on Amazon are best.

**Comment by nmcrae85** [comment]
anyone?

**Comment by colinhatfield** [comment]
I have a bunch of these and they work great:
https://www.amazon.co.uk/dp/B09B2WLRWX
https://www.amazon.co.uk/dp/B09B2NYJ3T
